### DISCUSSION TITLE
v9.0.0-beta4

### DISCUSSION THREAD

**Discussion by mivano** [serilog/serilog-sinks-elasticsearch#436] [Announcements]
## What's Changed
* Create codeql-analysis.yml by @mivano in https://github.com/serilog-contrib/serilog-sinks-elasticsearch/pull/370
* fix: Correct comment about default TypeName by @mikkelbu in https://github.com/serilog-contrib/serilog-sinks-elasticsearch/pull/393
* Proper handling of TypeName = null from appsettings.json by @mariwing in https://github.com/serilog-contrib/serilog-sinks-elasticsearch/pull/420
* Add `ElasticsearchSinkOptions.BufferFileRollingInterval` option by @andrey-kozlov-skuvault in https://github.com/serilog-contrib/serilog-sinks-elasticsearch/pull/416
* Fix internal or IntelliSense typos by @boflynn in https://github.com/serilog-contrib/serilog-sinks-elasticsearch/pull/406
* Bump actions/setup-dotnet from 1 to 2 by @dependabot in https://github.com/serilog-contrib/serilog-sinks-elasticsearch/pull/431
* Bump actions/checkout from 2 to 3 by @dependabot in https://github.com/serilog-contrib/serilog-sinks-elasticsearch/pull/427
* Bump actions/cache from 2 to 3 by @dependabot in https://github.com/serilog-contrib/serilog-sinks-elasticsearch/pull/429
* Bump actions/download-artifact from 2 to 3 by @dependabot in https://github.com/serilog-contrib/serilog-sinks-elasticsearch/pull/424

## New Contributors
* @mikkelbu made their first contribution in https://github.com/serilog-contrib/serilog-sinks-elasticsearch/pull/393
* @mariwing made their first contribution in https://github.com/serilog-contrib/serilog-sinks-elasticsearch/pull/420
* @andrey-kozlov-skuvault made their first contribution in https://github.com/serilog-contrib/serilog-sinks-elasticsearch/pull/416
* @boflynn made their first contribution in https://github.com/serilog-contrib/serilog-sinks-elasticsearch/pull/406
* @dependabot made their first contribution in https://github.com/serilog-contrib/serilog-sinks-elasticsearch/pull/431

**Full Changelog**: https://github.com/serilog-contrib/serilog-sinks-elasticsearch/compare/v8.4.1+4.build.444...v9.0.0-beta1

<hr /><em>This discussion was created from the release <a href='https://github.com/serilog-contrib/serilog-sinks-elasticsearch/releases/tag/v9.0.0-beta4'>v9.0.0-beta4</a>.</em>

**Comment by johnarsc** [comment]
Serilog.Sinks.Elasticsearch v9.0.0-beta4 requires Serilog.Formatting.Elasticsearch 9.0.0-beta4, but Serilog.Formatting.Elasticsearch 9.0.0-beta4 missing from nuget

**Reply by jadanah** [reply]
Same issue 9.0.0-beta4 is not available on nuget 

https://www.nuget.org/packages/Serilog.Formatting.Elasticsearch/9.0.0-beta4

**Reply by mivano** [reply]
Thanks, trouble with ownership of the packages at the moment. Workaround is to download/use them from GitHub Packages.

**Reply by mivano** [reply]
Sorry, took some time as I got no access in NuGet to this package. That is fixed now and the beta7 contains both packages on NuGet.
